### ISSUE TITLE
Is it possible to disable auto scroll ?

### ISSUE THREAD

**Issue by jseparovic** [josdejong/svelte-jsoneditor#79]
Is it possible to disable auto scroll?
When I have multiple editors on the page it always scrolls down the page to the middle one for some reason.
It would be good if auto scroll could be disabled entirely to let the user manually scroll from the top of the page.

I've updated the previous sandbox:
https://codesandbox.io/s/blue-cloud-i4vo7

Awesome tool btw. The tree view is great in my project. I really like the way you parse \n in strings too. I have yaml inside json and this works really well.

Also, I'm getting a strange issue where the expand method on load is not working on one of the editors which has a fair bit of data. I can't reproduce it in the codesandbox though which is strange.
<img width="280" alt="image" src="https://user-images.githubusercontent.com/4665758/168524052-870bcc36-f212-4b67-b7a9-ccbea7b2d7ec.png">

**Comment by jseparovic** [comment]
Actually, if I add that setTimeout workaround like you mentioned in the previous issue (with a small delay) it expands ok.
```
setTimeout(() => editor.current.expand(), 250);
```

**Comment by josdejong** [comment]
Thanks for reporting Jason, and for the sandbox that demonstrates the issue. It looks like a bug, probably related to the editors not having a fixed height (and scrolling contents). Will look into it.

> Awesome tool btw. The tree view is great in my project. I really like the way you parse \n in strings too. I have yaml inside json and this works really well.

Good to hear 😎

**Comment by jseparovic** [comment]
btw, I was hoping to disable the autoscroll altogether. I have an instance details view which shows every attribute of an inventory object, and if it has json attributes, I show the editor. The first editor could be a ways down the page, I was hoping to keep the scroll position on load set to the top of the page and not the first editor

**Comment by josdejong** [comment]
Yes definitely. Scrolling the editor into view was a bug. I found the cause: for the navigation bar I used some code to scroll to the right side when the path is longer than the width of the editor. I used `element.scrollIntoView` there, not realizing this side effect of scrolling the editor itself into view when it isn't. I've fixed it by changing to use `element.scrollTo`, see commit https://github.com/josdejong/svelte-jsoneditor/commit/42fe8188e9248a0191d39e8e991217d5dec5a54c. Will publish the fix today.
